DigitalOffice/Services/UserService/Mappers/UsersMapper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Linq;
+using UserService.Database.Entities;
+using UserService.Mappers.Interfaces;
+using UserService.Models;
+
+namespace UserService.Mappers
+{
+    public class UsersMapper : IMapper<DbUser, User>
+    {
+        public User Map(DbUser dbUser)
+        {
+            if (dbUser == null)
```

Comment:
Hm. Validator?

Reply:
dbUser is coming from database so no need to validate it

Reply:
I do not mean it. I was surprised that there is a if